README.md

```diff
@@ -212,16 +212,31 @@ Parameters:
+| Name (field.type)      | Input                                                                                        | Pulsar Schema Type | AVRO Schema Type            | Expression Examples                                                     |
+|------------------------|----------------------------------------------------------------------------------------------|--------------------|-----------------------------|-------------------------------------------------------------------------|
+| `STRING`               | A unicode character sequence.                                                                | `STRING`           | `string`                    | "'first name'", "fn:str(value)", "fn:concat(value, '-suffix')"          |
+| `INT8`                 | An 8-bit integer.                                                                            | `INT8`             | `int`                       | "127", "1 + 1"                                                          |
+| `INT16`                | A 16-bit integer.                                                                            | `INT16`            | `int`                       | "32768"                                                                 |
+| `INT32`                | A 32-bit integer.                                                                            | `INT32`            | `int`                       | "2147483647"                                                            |
+| `INT64`                | A 64-bit integer.                                                                            | `INT64`            | `int`                       | "9223372036854775807"                                                   |
+| `FLOAT`                | A 32-bit floating point.                                                                     | `FLOAT`            | `float`                     | "340282346638528859999999999999999999999.999999", "1.1 + 1.1"           |
+| `DOUBLE`               | A 64-bit floating point.                                                                     | `DOUBLE`           | `double`                    | "1.79769313486231570e+308"                                              |
+| `BOOLEAN`              | true or false                                                                                | `BOOLEAN`          | `boolean`                   | "true", "1 == 1", "value.stringField == 'matching string'"              |
+| `DATE`                 | A date/time object with an offset from UTC resembling the old Java 8 type.<br>               | `date`             | `date` (logical)            | "'2022-10-02T00:00:00Z'", "1664668800000" (millis), 19267 (days)        |
+|                        | The input can be a string in the RFC3339 format or number of millis since epoch.<br>         |                    |                             |                                                                         | 
```

Comment:
The input can now be a lot of things (Instant, Double, etc...).
We should remove these lines and have a small note after the table on how Strings are converted to timestamps.